magnite-batch

jakeseawell opened the pull request:
- Updated doc and images

jakeseawell committed:
magnite-batch

 - Updated doc and images

jakeseawell committed:
magnite-batch

 - Updated 'Magnite Streaming' to 'Magnite'

jakeseawell committed:
magnite-batch

 - Added missing DNL

nscullion commented on help/destinations/catalog/advertising/magnite-batch.md:
```diff
@@ -1,25 +1,29 @@
-To recap: If you only want to deliver Adobe Real-Time CDP audiences once per day, you will use the Magnite: Batch destination only, and audiences will be delivered once per day. If you want to deliver Adobe Real-Time CDP audiences in Real-Time, you will use *both* the Magnite: Batch destination, and the Magnite Real-Time destination. For more information, reach out to Magnite: Streaming.
+2. However, if you want or need to deliver audiences more frequently, you will need to use the [Magnite: Real-Time destination](/help/destinations/catalog/advertising/magnite-streaming.md). When using the real-time destination, [!DNL Magnite] will receive audiences in real-time, but [!DNL Magnite] can only store real-time audiences temporarily in their platform, and they will be removed from the  ...
 
+To recap: If you only want to deliver Adobe Real-Time CDP audiences once per day, you will use the [!DNL Magnite: Batch] destination only, and audiences will be delivered once per day. If you want to deliver Adobe Real-Time CDP audiences in real-time, you will use *both* the [!DNL Magnite: Batch] destination, and the[!DNL Magnite: Real-Time] destination. For more information, reach out to [!DNL M ...
```
`To recap: If you only want to deliver Adobe **Real-Time** CDP audiences once per day, you will use the [!DNL Magnite: Batch] destination only`
think this should be batch?

jakeseawell replied:
"Adobe Real-Time CDP audiences can be delivered to the Magnite platform in two ways - they can be delivered once per day, or they can be delivered in real-time:"

Not sure what `Adobe Real-Time CDP audiences` are.. Should we just be calling these `Adobe CDP audiences`?

nscullion replied:
Yea that's a better catch! I think this should be `Adobe CDP audiences`

jakeseawell replied:
updated

jakeseawell committed:
magnite-batch

 - Updated wording

jakeseawell committed:
magnite-batch

 - Updated 'Adobe Real-Time CDP audeinces' to 'Adobe CDP audeinces'

nscullion reviewed (approved):
Looks great! Nicely done!

jakeseawell committed:
magnite-batch

 - Updated magnite-batch.md doc
 - Updated the screenshots for audience-field-mapping and audience-select-device-type
